src/Serilog.Sinks.OpenTelemetry/Helpers/OpenTelemetryEnvironment.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Serilog.Sinks.OpenTelemetry;
+
+namespace Serilog.Helpers;
```

Comment:
Since this is OpenTelemetry-related, it can go in `Serilog.Sinks.OpenTelemetry.*`, perhaps under an `Configuration` namespace?